framework/src/controller/schema/application.js

```diff
@@ -211,24 +211,47 @@ module.exports = {
 				properties: {
 					nethash: {
 						type: 'string',
+						format: 'hex',
```

Comment:
I'm not sure about this... Why `hex` @shuse2? What you can add here is the length `64`.

Reply:
`hex` formats for /^[a-f0-9]+$/`, I thought it also checked for length `64` but apparently it doesn't.